### PR TITLE
Update queries.md

### PR DESCRIPTION
**Pull request by castillo92**
Completing some examples to allow newbie people to understand relative time and time shift time

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by achatterjee-grafana — APPROVED** [review]
Looks good, developers need to chime in on syntax usage.

File: docs/sources/panels/queries.md [hunk 1]
```diff
@@ -101,6 +101,32 @@ Panel data source query options:
...
+| This week so far    | `now/w`             |
+| Last 2 years        | `now-2y/y`          |   
+
+
+- **Time shift:** 
+
+| Example             | Time shift field |
+| ------------------- | --------------------|
+| Last entire week    | `1w/w`              |
+| Two entire week ago | `2w/w`              |
```
Comment (geshii) [inline docs/sources/panels/queries.md:122]:
```suggestion
| Two entire weeks ago | `2w/w`              |
```

**Review by vmfunc — CHANGES_REQUESTED** [review]
typo

**Comment by castillo92** [comment]
> [CLA assistant check](https://cla-assistant.io/grafana/grafana?pullRequest=31941)
> All committers have signed the CLA.

Signed! I forgot do that

**Comment by ying-jeanne** [comment]
LGTM @castillo92 you probably need to do a rebase on main to make the CI pass.

**Review by ying-jeanne — APPROVED** [review]

**Comment by castillo92** [comment]
> LGTM @castillo92 you probably need to do a rebase on main to make the CI pass.

Could you please explain me how to do that?

**Pull request merged**